boards/nordic/nrf52840_chip_layout.ld

```diff
@@ -5,8 +5,8 @@
 /* Memory Space Definitions, 1M flash, 256K ram */
 MEMORY
 {
-  rom (rx)  : ORIGIN = 0x00000000, LENGTH = 192K
-  prog (rx) : ORIGIN = 0x00030000, LENGTH = 832K
+  rom (rx)  : ORIGIN = 0x00000000, LENGTH = 256K
+  prog (rx) : ORIGIN = 0x00030000, LENGTH = 768K
```

Comment:
I'm 99% certain that Amit reverted this. It made apps not work. And it's not in: https://github.com/tock/tock/compare/master...dev/software-hmac-integration

Reply:
Oh, you addressed this on Slack. As long as it's intentional and apps still work, I'm fine with it.